core/src/apps/cardano/shelley/__init__.py

```diff
@@ -0,0 +1,5 @@
+from apps.common import HARDENED
+
+CURVE = "ed25519"
+SHELLEY_SEED_NAMESPACE = [HARDENED | 1852, HARDENED | 1815]
+BYRON_SEED_NAMESPACE = [HARDENED | 44, HARDENED | 1815]
```

Comment:
Perhaps 1852 | HARDENED (we use it like this elsewhere).

Reply:
It's used in both ways throughout the project. However I've swapped it, it seems more readable. I also made a note to swap it in the tests, because even byron tests use this ordering. Also the tests should have a constant - made a note about that too.